langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -104,6 +143,13 @@ public isolated function 'join(string separator, string... strs) returns string
 
 # Finds the first occurrence of one string in another string.
 #
+# ```ballerina
+# "New Zealand".indexOf("land") ⇒ 7
+#
+# // Search for the first occurrence of a string from a specific index onwards.
```

Comment:
Do we need these comments? Along with rest of the documentation won't it be self-explanatory? We removed the comments in the array langlib btw.

Reply:
Yes, I'll remove this since it is already covered `return` description.